TEMPLATES/.cfnnag

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Here we're making public a list of rules we want to internally suppress. Will this be a problem?

Reply:
Yes may be. Specially this line 
`many existing projects are not set up for key rotation right now.` 
If there is way in GitHub, just to make single file private even if repo is public?